myStyles.css

```diff
@@ -232,10 +232,34 @@
   height: 29%;
 }
 
+.bandImgCont {
+  margin-right: 5%;
+  position: relative;
+  width: 33%;
+  max-width: 130px;
+  max-height: 130px;
+}
+
 .bandImgCont img {
-  width: 100%;
-  height: 150px;
+  /* width: 150px;
+  height: 150px; */
   object-fit: scale-down;
+  position: absolute;
+  /* top: 0;
+  left: 0; */
+}
```

Comment:
Kindly take away every commented piece of code in your project as it makes your code base look messy and gives it a less professional look😉.